scripts/llnl_scripts/llnl_lc_build_tools.py

```diff
@@ -492,6 +492,8 @@ def full_build_and_test_of_tpls(builds_dir, timestamp, spec, report_to_stdout =
     if not os.path.exists(prefix):
         os.mkdir(prefix)
     prefix = pjoin(prefix, timestamp)
+    if not os.path.exists(prefix):
```

Comment:
This intermittently failed when it went to write the file on the now line 501. Not sure why sometimes this is okay but others not. This did make my problem go away.